packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -143,6 +151,11 @@ export abstract class QueueProcessingServiceBase extends Construct {
    */
   public readonly sqsQueue: IQueue;
 
+  /**
+   * The dead letter queue for the SQS queue that the service will process from
```

Comment:
Not sure if it makes sense to say the service will process from, might want to rephrase this as `The dead letter queue for the SQS queue`